Dockerfile

```diff
@@ -12,43 +12,34 @@
 # 
 # docker build  --build-arg BUILD_REPO=YourFork/PokemonGo-Bot --build-arg BUILD_BRANCH=6a4580f .
 
-FROM python:2.7.12-alpine
+FROM alpine
```

Comment:
alpine after apk add python is about 52.2 MB instead of 72MB for python:2.7-alpine
the biggest saving is from executing api add and api del in the same RUN stanza